railties/lib/rails/commands/console/console_command.rb

```diff
@@ -98,7 +98,9 @@ def perform
         # RAILS_ENV needs to be set before config/application is required.
         ENV["RAILS_ENV"] = options[:environment]
 
-        require_application_and_environment!
```

Comment:
So this method is still used by other commands like `destroy` and `help`.
These command will not log to stdout?

Reply:
The reason I only included `server` and `console` is those are the only commands that are broadcasted to `STDOUT` right now. However, I definitely agree that users could expect other commands to output to `STDOUT` as well. Maybe the answer would be to _always_ broadcast to `STDOUT` in development?

That would also have the benefit of simplifying this change as it wouldn't need the `log_to_stdout` config at all.